crates/fibre/src/lib.rs

```diff
@@ -60,36 +163,25 @@ impl FiberStack {
     }
 }
 
-pub struct Fiber<'a, Resume, Yield, Return> {
+pub struct Fiber {
     stack: FiberStack,
     inner: imp::Fiber,
     done: Cell<bool>,
-    _phantom: PhantomData<&'a (Resume, Yield, Return)>,
+    _phantom: PhantomData<()>,
```

Comment:
This is redundant now, I think. `PhantomData` ought only be necessary when you want to constrain the lifetime of unused type parameters.

Reply:
Done